alfresco-transformer-base/src/main/java/org/alfresco/transformer/messaging/MessagingInfo.java

```diff
@@ -0,0 +1,30 @@
+package org.alfresco.transformer.messaging;
```

Comment:
Please also add licence header (as per other Java files), thanks.

Reply:
Yes, I forgot about that (new IDE, new settings).
Done.